client/src/components/CheckoutOptions.jsx

```diff
@@ -1,20 +1,69 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable no-useless-constructor */
+/* eslint-disable class-methods-use-this */
```

Comment:
what do these rules enforce? (trying to figure out what benefit we might get from disabling them)